#Camera topic here
camera_topic: "/img_for_sort"
queue_size: 1

# Set your detection topic here
detection_topic: "/detected_objects_in_image"

# Set your output topic here
tracker_topic: "/sort_track_deep"

model_dir: "/home/danmuzi/catkin_ws/src/sort-deepsort-yolov3-ROS/sort_track/src/deep_sort/model_data/mars-small128.pb"
image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true
